### ISSUE TITLE
This list includes valid e-mail services, and is causing people to not receive important communication (bills, receipts, etc)

### ISSUE THREAD

**Issue by inaun** [ivolo/disposable-email-domains#979]
I'm not sure what is wrong with you people. This list contains domains used by at least one popular and valid e-mail service. People like me use this service for long-term communications. The help forums for this service are full of people complaining about lost bills, lost receipts, and other important communications that are being dumped.

I personally cannot receive health-related communications from one of my doctors!

The service I'm talking about is Spam Gourmet. This service is used for long-term communications. It provides the ability to allow wanted e-mail through while rejecting unwanted email. In other words, it is an anti-spam service -- NOT a disposable e-mail service!

The spamgourmet.com domain needs to be IMMEDIATELY removed from this list, so important communications (like health communications, bills, receipts, shipping updates, etc) are not blocked by web developers who rely on this list. Further, this list needs to be properly maintained so valid e-mail services are not put onto the list.

**Comment by domainfun** [comment]
@inaun  I'm sorry to hear that your email service provider has had a false positive, i.e. that they rejected a genuine email from your doctor to you.

However, [ticket #979](https://github.com/ivolo/disposable-email-domains/issues/979) is a duplicate.  Before opening a new issue, you should have looked at existing issues - a quick glance at https://github.com/ivolo/disposable-email-domains/issues shows that [ticket #933](https://github.com/ivolo/disposable-email-domains/issues/933) already mentions `spamgourmet`.

Secondly, the repository [disposable-email-domains](https://github.com/ivolo/disposable-email-domains) is provided free of charge, and no one is forcing you or your email service provider to use this repository or to reject messages addressed to you just because `spamgourmet.com` is listed in this piece of code.

Please contact your email service provider and ask them to skip their spam filter check against the domain list offered on `disposable-email-domains` for email senders using `spamgourmet.com`.  Or they could use another list of disposable mail domains instead, such as the list published on [unkn0w/disposable-email-domain-list](https://github.com/unkn0w/disposable-email-domain-list).

By the way, you email provider should know that the list offered at `disposable-email-domains` is NOT perfect - the licence clearly says:

THE SOFTWARE IS PROVIDED "AS IS" AND "AS AND WHEN AVAILABLE", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.

PS: I do agree that mail domain `spamgourmet.com` should be removed from this list, but having been a commercial email service provider for over 15 years, I'm well aware that using free/open-source blacklists or whitelists for filtering incoming emails does require both an additional local blacklist and a local whitelist, in order to provide a tailored approach according to personal preferences and (business) use-cases.

Oh, and I personally would NOT trust the `spamgourmet.com` service, as they don't even run a secure DNS service for their domain name (`DNSSEC: unsigned`):

```
$ whois spamgourmet.com
   Domain Name: SPAMGOURMET.COM
   Registry Domain ID: 28777446_DOMAIN_COM-VRSN
   Registrar WHOIS Server: whois.namecheap.com
   Registrar URL: http://www.namecheap.com
   Updated Date: 2023-03-01T04:22:52Z
   Creation Date: 2000-06-07T19:10:38Z
   Registry Expiry Date: 2026-06-07T19:10:38Z
   Registrar: NameCheap, Inc.
   Registrar IANA ID: 1068
   Registrar Abuse Contact Email: abuse@namecheap.com
   Registrar Abuse Contact Phone: +1.6613102107
   Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
   Name Server: DNS1.REGISTRAR-SERVERS.COM
   Name Server: DNS2.REGISTRAR-SERVERS.COM
   DNSSEC: unsigned
   URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of whois database: 2024-01-04T02:14:36Z <<<
```

**Comment by eventcountdown123-rgb** [comment]
Hey, I ran into this same issue with spam signups recently. The problem is that simple Regex validation passes most disposable domains because they have valid MX records.

I ended up building a small API to check domains against a live blocklist of ~60k burner providers. It filters out the subtle ones like `xy7-temp.com` that usually slip through.

It solved the issue for me, so I opened up the API for others to use. You can check it out here if you want to test it: [https://vishodi.com/](https://vishodi.com/disposable-email-detection) (There's a free tier for devs).

Hope it helps!
